### PR TITLE
Update mbed_app.json

### PR DESCRIPTION
**Pull request by TeemuKultala**

### CODE REVIEW

**Comment by adbridge** [comment]
@TeemuKultala As this is a fix of OOB can you please direct this to the 5.7-oob branch? The usual process is that during OOB any issues found are fixed on the OOB branch and then at then end all fixes are pushed back up to master along with the published version of mbed lib.

**Comment by MarceloSalazar** [comment]
Please send PR to this branch: https://github.com/ARMmbed/mbed-os-example-cellular/tree/mbed-os-5.7.0-oob

File: mbed-os.lib [hunk 1]
```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/mbed-os/#5f6572179d66ce4c09d6517b659ac51133cc980d
+https://github.com/ARMmbed/mbed-os/#da9464d103057f63064e49df258c1e5ca901c07c
```
Comment (MarceloSalazar) [inline mbed-os.lib]: Why are you updating Mbed OS lib? There will be RC2 soon overriding this.
FYI @adbridge

Comment (MarceloSalazar) [reply mbed-os.lib]: I see this has been updated to oob-5.7.0-RC1, so that's fine :)

**Comment by adbridge** [comment]
May have been due to his merge...

**Pull request merged**